internal/service/verify/resource_verify_voice_phrase_content.go

```diff
@@ -0,0 +1,395 @@
+package verify
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/patrickcping/pingone-go-sdk-v2/pingone/model"
+	"github.com/patrickcping/pingone-go-sdk-v2/verify"
+	"github.com/pingidentity/terraform-provider-pingone/internal/framework"
+	"github.com/pingidentity/terraform-provider-pingone/internal/sdk"
+	validation "github.com/pingidentity/terraform-provider-pingone/internal/verify"
+)
+
+// Types
+type VoicePhraseContentResource struct {
+	client *verify.APIClient
+	region model.RegionMapping
+}
+
+type voicePhraseContentResourceModel struct {
+	Id            types.String `tfsdk:"id"`
+	EnvironmentId types.String `tfsdk:"environment_id"`
+	VoicePhraseId types.String `tfsdk:"voice_phrase_id"`
+	Locale        types.String `tfsdk:"locale"`
+	Content       types.String `tfsdk:"content"`
+	CreatedAt     types.String `tfsdk:"created_at"`
+	UpdatedAt     types.String `tfsdk:"updated_at"`
+}
+
+// Framework interfaces
+var (
+	_ resource.Resource                = &VoicePhraseContentResource{}
+	_ resource.ResourceWithConfigure   = &VoicePhraseContentResource{}
+	_ resource.ResourceWithImportState = &VoicePhraseContentResource{}
+)
+
+// New Object
+func NewVoicePhraseContentResource() resource.Resource {
+	return &VoicePhraseContentResource{}
+}
+
+// Metadata
+func (r *VoicePhraseContentResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_verify_voice_phrase_content"
+}
+
+func (r *VoicePhraseContentResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+
+	// schema descriptions and validation settings
+	const attrMinLength = 1
+
+	// P1 Platform does not set a traditional UUID for its default phrase IDs
+	const defaultVoicePhraseIds = "(exceptional_experiences|pingone_davinci_nocode)"
```

Comment:
Removed. Configuring the default phrases in the provider is not allowed because the phrases ids are non-standard, simply examples, and may change in the future.

Disallowing the implementer from using the pre-loaded example phrases (which are product-focused, and may change in the future) prevents reliance on non-standard UUIDs and configuration objects that may change in the future outside of the implementer's control.